src/components/Home/index.js

```diff
@@ -1,27 +1,45 @@
 import React, { Component } from 'react';
 
-import { connect } from 'react-redux'
-import { compose } from 'redux'
-import { firebaseConnect } from 'react-redux-firebase'
-import withAuthorization from '../Auth/withAuthorization'
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { firebaseConnect } from 'react-redux-firebase';
+import withAuthorization from '../Auth/withAuthorization';
+import Invoices from '../Invoice/invoices';
+import NewInvoice from '../Invoice/newInvoice';
+import { Grid } from 'semantic-ui-react';
+
+import './index.css';
+
 
 class Home extends Component {
   render() {
-    return (<div>home, home, home</div>);
+    const { documents, firebase } = this.props;
+
+    return (
+      <Grid>
+        <Grid.Row>
+          <Grid.Column floated='right' width={2}>
```

Comment:
Why do we pass the firebase instance throughout the components. Surely this is more like a singleton that can be loaded from a module when needed?